_config.yml

```diff
@@ -30,6 +30,73 @@ defaults:
 # Aliases available in the markdown template. The below will expand {{site.foo}} to Bar in the output
 # "foo": "Bar"
 
+# Telerik products and trademarks
+ab: "AppBuilder"
+tab: "Telerik AppBuilder"
+bs: "Backend Services"
+tbs: "Telerik Backend Services"
+am: "AppManager"
+tam: "Telerik AppManager"
+eq: "Analytics"
+teq: "Telerik Analytics"
+af: "Feedback"
+taf: "Telerik AppFeedback"
+tp: "Telerik Platform"
+sb: "Screen Builder"
+tsb: "Telerik Screen Builder"
+ns: "NativeScript"
+tns: "Telerik NativeScript"
+vpm: "Verified Plugins Marketplace"
+tvpm: "Telerik Verified Plugins Marketplace"
+tjc: "Telerik JustCode"
+kui: "Kendo UI"
+tkui: "Telerik Kendo UI"
+
+# Telerik UI services
+
+ab-s: "Code"
+sb-s: "Views"
+bs-d: "Data"
+bs-u: "Users"
+bs-n: "Notifications"
+bs-bl: "Business Logic"
+eq-s: "Analytics"
+af-s: "Feedback"
+
+# Telerik components
+
+dc: data connector
+dcs: data connectors
+dct: Data Connectors
+dls: Data Link Server
+
+# 3rd party products and trademarks
+ac: "Apache Cordova"
+wp: "Windows Phone"
+wp8: "Windows Phone 8"
+wp81: "Windows Phone 8.1"
+wp-store: "Windows Phone Store"
+ios-store: "App Store"
+android-store: "Google Play"
+
+# Other
+# Style note: Use a human-readable variable name.
+# Ex. good: EverliveURLSecure: https://api.everlive.com/v1
+# Ex. bad: ELS: https://api.everlive.com/v1
+AppId: "your-app-id"
```

Comment:
Perhaps app-id instead of AppId and master-key instead of MasterKey. This way we'll stick to the hyphen convention from the entries above.